app/utils/cu-profiling.ts

```diff
@@ -1,10 +1,29 @@
+import { ComputeBudgetProgram, ParsedInstruction, PartiallyDecodedInstruction, PublicKey } from '@solana/web3.js';
+import { ComputeBudgetInstruction, identifyComputeBudgetInstruction } from '@solana-program/compute-budget';
+import {
+    AssociatedTokenInstruction,
+    CREATE_ASSOCIATED_TOKEN_DISCRIMINATOR,
+    identifyAssociatedTokenInstruction,
+    identifyTokenInstruction,
+    TokenInstruction,
+} from '@solana-program/token';
+import bs58 from 'bs58';
+
+import { camelToTitleCase } from '.';
 import { Cluster } from './cluster';
 import { getReservedComputeUnits } from './compute-units-schedule';
 import { getDefaultComputeUnits } from './default-compute-units';
 import { InstructionLogs } from './program-logs';
+import { getProgramName } from './tx';
+
+const ASSOCIATED_TOKEN_PROGRAM_ID = new PublicKey('ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL');
+const TOKEN_PROGRAM_ID = new PublicKey('TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA');
+const TOKEN_2022_PROGRAM_ID = new PublicKey('TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb');
+const MEMO_PROGRAM_ID = new PublicKey('MemoSq4gqABAXKb96qnH8TysNcWxMyWCqXgDLGmfcHr');
```

Comment:
Clients from `@solana-program/<%program_name%>` have these names exported. I'd suggest using them instead of hardcoding the addresses